readme.md

```diff
@@ -59,6 +59,13 @@ Default: `true`
 
 Overwrite existing destination file.
 
+##### directoryMode
+
+Type: `number` | `string`\
```

Comment:
I would prefer to only document the number support. The string support is only in Node.js for legacy reasons.